django/contrib/admin/static/admin/js/jquery.init.js

```diff
@@ -1,8 +1,8 @@
 /*global django:true, jQuery:false*/
 /* Puts the included jQuery into our own namespace using noConflict and passing
  * it 'true'. This ensures that the included jQuery doesn't pollute the global
- * namespace (i.e. this preserves pre-existing values for both window.$ and
- * window.jQuery).
+ * namespace (i.e. this preserves pre-existing values for window.$ if window.$
+ * is not a jQuery object, but removes jQuery from the global namespace).
```

Comment:
It seems like jQuery.noConflict will keep jQuery in the global namespace if another copy of jQuery is included before Django's:
> If for some reason two versions of jQuery are loaded (which is not recommended), calling $.noConflict( true ) from the second version will return the globally scoped jQuery variables to those of the first version.

Maybe the comment means "this preserves pre-existing values for both window.$ and window.jQuery _if another copy of jQuery is loaded before Django's copy_"?